### ISSUE TITLE
One page fixed-sidebar docs?

### ISSUE THREAD

**Issue by tmcw** [Turfjs/turf-www#62]
This is something of an opinionated change. @samanpwbb suggested something like the [underscore](http://underscorejs.org/) layout.

To make this happen I could write a lazy-evaluation mode for rpl

**Comment by morganherlocker** [comment]
underscore's doc accessibility is lacking IMO. It relies on users [cmd]+[f]ing things (which many do not know how to use effectively), and ignores users who don't want to download loads of text at once (low bandwidth, small cpu, and screen-readers for the visually impaired especially). 

Linking with text anchors is weird too, and will be difficult for many people. lodash even added this cryptic collection of icons to allow people to link more easily:

![](https://dl.dropbox.com/s/xjttefshoylqxil/Screenshot%202015-02-05%2014.32.59.png?dl=0)

The fact that a `#` button is even necessary seems to point to the fact that many people wanted to link to things, but couldn't figure out how.

\2cents

**Comment by tmcw** [comment]
Well, we're halfway here: turf docs are now fixed-sidebar, but not one-pages.

**Comment by morganherlocker** [comment]
Happy medium here? (reopen if not)
